masonry/src/widget/tests/safety_rails.rs

```diff
@@ -80,6 +80,7 @@ fn check_forget_to_recurse_widget_added() {
 #[should_panic(expected = "not visited in method layout")]
 #[test]
 fn check_forget_to_recurse_layout() {
+    assert!(cfg!(debug_assertions));
```

Comment:
This is incorrect: you can run unit tests in release mode.

This should probably be an early exit instead.

Reply:
An early exit isn't possible here, because this is a `should_panic` test.

Reply:
Oooh! That's annoying. I guess then we should either:

- `#[cfg]` the test.
- Create a synthetic panic in release mode.
- Use `catch_unwind` instead.

In any case, you don't need to solve this now, but a TODO comment would be nice.

Reply:
Created #477 for this, you can link to it.